spec/features/campaign_spec.rb

```diff
@@ -30,13 +29,12 @@
       click_button 'Create Campaign'
 
       # Validations
-      expect(flash).
-          to eq('Campaign successfully created.')
+      expect(flash).to eq('Campaign successfully created.')
 
-      expect(find('#name_1').text).to eq('Test Campaign')
-      expect(find('#visits_1').text).to eq('0')
-      expect(find('#registrations_1').text).to eq('0')
-      expect(find('#submissions_1').text).to eq('0')
+      expect(page).to have_selector('.campaign-name', text: 'Test Campaign')
```

Comment:
Can't we do this using the ID instead of introducing a class we don't need anywhere else except for the test?